functional-samples/tutorial.websockets/README.md

```diff
@@ -0,0 +1,12 @@
+# Chrome Extension WebSocket Example
+
+This example demonstrates how to use WebSockets in a MV3 Chrome Extension starting with Chrome version M116.
+
+## Running this extension
+
+1. Clone this repository.
```

Comment:
From what I can tell, if the Glitch app goes to sleep you have to wake it over HTTP before you can connect on a WebSocket. Can we add a note about that (e.g visit this HTTP URL and then wait a few seconds) before trying the sample?